specification/cluster/health/ClusterHealthResponse.ts

```diff
@@ -27,38 +27,46 @@ import { IndexHealthStats } from './types'
  * @doc_id cluster-health
  */
 export class Response {
-  body: {
-    /** The number of active primary shards. */
-    active_primary_shards: integer
-    /** The total number of active primary and replica shards. */
-    active_shards: integer
-    /** The ratio of active shards in the cluster expressed as a percentage. */
-    active_shards_percent_as_number: Percentage
-    /** The name of the cluster. */
-    cluster_name: Name
-    /** The number of shards whose allocation has been delayed by the timeout settings. */
-    delayed_unassigned_shards: integer
-    indices?: Dictionary<IndexName, IndexHealthStats>
-    /** The number of shards that are under initialization. */
-    initializing_shards: integer
-    /** The number of nodes that are dedicated data nodes. */
-    number_of_data_nodes: integer
-    /** The number of unfinished fetches. */
-    number_of_in_flight_fetch: integer
-    /** The number of nodes within the cluster. */
-    number_of_nodes: integer
-    /** The number of cluster-level changes that have not yet been executed. */
-    number_of_pending_tasks: integer
-    /** The number of shards that are under relocation. */
-    relocating_shards: integer
-    status: HealthStatus
-    /** The time since the earliest initiated task is waiting for being performed. */
-    task_max_waiting_in_queue?: Duration
-    /** The time expressed in milliseconds since the earliest initiated task is waiting for being performed. */
-    task_max_waiting_in_queue_millis: DurationValue<UnitMillis>
-    /** If false the response returned within the period of time that is specified by the timeout parameter (30s by default) */
-    timed_out: boolean
-    /** The number of shards that are not allocated. */
-    unassigned_shards: integer
-  }
+  body: ResponseBody
```

Comment:
Why moving it to a different type? If there's a good reason for this, we should at least name it with a more meaningful `HealthResponseBody`.

Reply:
Doh, I did not see it's also used for the 408 response. Naming still applies ;-)